metadata-ingestion/docs/sources/qlik-sense/qlik-sense_recipe.yml

```diff
@@ -0,0 +1,25 @@
+source:
+  type: qlik-sense
+  config:
+    # Coordinates
+    tenant_hostname: "xyz12xz.us.qlikcloud.com"
+    # Credentials
+    api_key: "QLIK_API_KEY"
+    
+    # Optional - filter for certain space names instead of ingesting everything.
+    # Mention 'personal_space' if entities of personal space need to ingest
```

Comment:
if there's hierarchy e.g. "My Space -> Nested Space -> App", does full path need to be specified?

Reply:
This filter just works at space level. And we can't have nested spaces.